docker/composer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer/composer
```

Comment:
Why would the container fail if an extension doesn't exist? Running it in an container basically ensures that it does.

I don't like having containers in the compose file that'll just clutter the log and then exit without doing anything.